unstructured/ingest/cli/cmds/weaviate.py

```diff
@@ -0,0 +1,73 @@
+import typing as t
+from dataclasses import dataclass
+
+import click
+
+from unstructured.ingest.cli.interfaces import CliConfig, Dict
+
+CMD_NAME = "weaviate"
+
+
+@dataclass
+class WeaviateCliConfig(CliConfig):
```

Comment:
Would make sense to import the configs this represents and extend them directly to avoid redefining he dataclass fields: 
```suggestion
from unstructured.ingest.connector.weaviate import SimpleWeaviateConfig
class WeaviateCliConfig(SimpleWeaviateConfig, CliConfig):
...
```